FakebookNotifications/FakebookNotifications.DataAccess/Models/Notification.cs

```diff
@@ -0,0 +1,33 @@
+﻿using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace FakebookNotifications.DataAccess.Models
+{
+    public class Notification
+    {
+        //mongo object id
+        [BsonId]
+        [BsonRepresentation(BsonType.ObjectId)]
+        public string Id { get; set; }
+
+        //notification type: post/comment/tag
+        [BsonElement("type")]
+        public string Type { get; set; }
+
+        //user logged in to receive notifications
+        [BsonElement("user")]
+        public User LoggedInUser { get; set; }
+
+        //user that triggered a notifiaction by commenting on a user post or tagging a user etc.
+        [BsonElement("trigger")]
+        public User TriggerUser { get; set; }
+
+        //if notification has been read
+        [BsonElement("read")]
+        public bool HasBeenRead { get; set; }
+
+        //date of notification
+        [BsonElement("date")]
+        public BsonDateTime Date { get; set; }
```

Comment:
keep in mind we want to store this data such that it's efficient to do things like "get the most recent 20 notifications for user X", so we probably want an index on Date. figure out how to configure that when the time comes